MAINTAINERS.md

```diff
@@ -1,144 +1,11 @@
-
-> This is great work! Based on your valuable contribution and ongoing engagement with the project, the current maintainers invite you to become a co-maintainer for this project. Please respond and let us know if you accept the invitation to become maintainer.
-
-Individuals accept the nomination by replying, or commenting, for example _"Thank you! I would love to."_
-
-### Addition
-
-Upon receiving three positive (+1) maintainer votes, and no vetoes (-1), from other maintainers, and after having privately confirmed interest with the nominee, the maintainer opens a pull request adding the proposed co-maintainer to MAINTAINERS.md. The pull request is approved and merged.
-
-> _Content from the above nomination._
-> 
-> The maintainers have voted and agreed to this nomination.
-
-The repo admin adjusts the new maintainer’s permissions accordingly, and merges the pull request.
-
-## Removing a Maintainer
-
-Removing a maintainer is a disruptive action that the community of maintainers should not undertake lightly. There are several reasons a maintainer will be removed from the project, such as violating the [code of conduct](https://github.com/opensearch-project/.github/blob/main/CODE_OF_CONDUCT.md), or taking other actions that negatively impact the project.
-
-### Moving On
-
-There are plenty of reasons that might cause someone to want to take a step back or even a hiatus from a project. Existing maintainers can choose to leave the project at any time, with or without reason, by making a pull request to move themselves to the "Emeritus" section of MAINTAINERS.md, and asking an admin to remove their permissions.
-
-### Inactivity
-
-Maintainer status never expires. If a maintainer becomes inactive for a time (usually several months), the repo admin may revoke maintainer level access to the repository for security reasons. A maintainer can reach out to the repo admin to get their permissions reinstated.
-
-If the repo is left without any maintainers, either by maintainer inactivity or moving on, the repo is considered unmaintained. The repo admin will seek out new maintainers and note the maintenance status in the repo README file.
-
-### Negative Impact on the Project
-
-Actions that negatively impact the project will be handled by the admins, in coordination with other maintainers, in balance with the urgency of the issue. Examples would be [Code of Conduct](CODE_OF_CONDUCT.md) violations, deliberate harmful or malicious actions, and security risks.
+| Maintainer              | GitHub ID                                   | Affiliation |
+| ----------------------- | ------------------------------------------- | ----------- |
+| Shivam Dhar             | [Shivamdhar](https://github.com/Shivamdhar) | Amazon      |
```

Comment:
If you could update, it will be great. Nevertheless, i can make new PR too.
```suggestion
| Junqiu Lei               | [junqiu-lei](https://github.com/junqiu-lei) |  Amazon   |
```